CHANGELOG_PENDING.md

```diff
@@ -12,10 +12,17 @@ BREAKING CHANGES:
   * [rpc] \#2298 `/abci_query` takes `prove` argument instead of `trusted` and switches the default
     behaviour to `prove=false`
   * [privval] \#2459 Split `SocketPVMsg`s implementations into Request and Response, where the Response may contain a error message (returned by the remote signer)
+  * [genesis] \#2565 `consensus_params.evidence_params.max_age` is now `time.Duration` (nanosecond count)
+        ```json
+        "evidence_params": {
+          "max_age": "172800000000000"
```

Comment:
oof. Is there not some way to make this more human readable?

Reply:
https://stackoverflow.com/a/48051946/820520